addons/l10n_at/data/template/account.account-at.csv

```diff
@@ -61,24 +72,29 @@
-"chart_at_template_2000","Forderungen von Partnern im Inland ohne eigenes Debitorenkonto","2000","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
-"chart_at_template_2099","Forderungen von Partnern (Point Of Sale)","2099","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_1800","Geleistete Anzahlungen auf Vorräte 20 %","1800","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_1801","Geleistete Anzahlungen auf Vorräte 10 %","1801","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_1803","Geleistete Anzahlungen auf Vorräte 0 %","1803","False","asset_prepayments","l10n_at.account_tag_l10n_at_ABI5"
+"chart_at_template_2000","Forderungen aus Lieferungen und Leistungen Inland","2000","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2099","Forderungen aus Lieferungen und Leistungen Inland (Point Of Sale)","2099","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2080","Einzelwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Inland","2080","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2090","Pauschalwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Inland","2090","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
-"chart_at_template_2100","Forderungen von Partnern im EU Raum ohne eigenes Debitorenkonto","2100","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2100","Forderungen aus Lieferungen und Leistungen Währungsunion","2100","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2130","Einzelwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Währungsunion","2130","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2140","Pauschalwertberichtigungen zu Forderungen aus Lieferungen und Leistungen Währungsunion","2140","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
-"chart_at_template_2150","Forderungen von Partnern in Drittstaaten ohne eigenes Debitorenkonto","2150","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2150","Forderungen aus Lieferungen und Leistungen sonstiges Ausland","2150","True","asset_receivable","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2180","Einzelwertberichtigungen zu Forderungen aus Lieferungen und Leistungen sonstiges Ausland","2180","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
 "chart_at_template_2190","Pauschalwertberichtigungen zu Forderungen aus Lieferungen und Leistungen sonstiges Ausland","2190","False","asset_current","l10n_at.account_tag_l10n_at_ABII1"
+"chart_at_template_2200","Forderungen gegenüber verbundenen Unternehmen","2200","False","asset_current","l10n_at.account_tag_l10n_at_ABII2"
```

Comment:
Normally you would create an own account for each of those, but as a generic placeholder, it is ok. I wonder what will happen during an upgrade/update in case this account code is already taken...same for all new placeholder accounts.